data/yaml/joint.yaml

```diff
@@ -324,6 +324,9 @@ corbon:
 cosli:
 - W13-07
 crac:
+  2022:
```

Comment:
@mjpost added codi-crac and codi to crac, relevant to https://github.com/acl-org/acl-anthology/pull/2187